packages/sdk/src/sdk/api/tracks/TrackUploadHelper.ts

```diff
@@ -66,11 +66,13 @@ export class TrackUploadHelper extends BaseAPI {
       ...trackMetadata,
       trackSegments: [],
       trackCid: audioResponse.results['320'],
-      previewCid: trackMetadata.previewStartSeconds
-        ? audioResponse.results[
-            `320_preview|${trackMetadata.previewStartSeconds}`
-          ]
-        : trackMetadata.previewCid,
+      previewCid:
+        trackMetadata.previewStartSeconds !== undefined &&
```

Comment:
I both appreciate the explicitness of this and also recommend just using `!= null` :-D